common/src/main/java/com/wynntils/screens/settings/WynntilsBookSettingsScreen.java

```diff
@@ -74,7 +74,7 @@ public static Screen create() {
 
     @Override
     protected void doInit() {
-        reloadConfigButtons();
+        reloadConfigurableButtons();
```

Comment:
I don't understand what this change does, neither the `selected` stuff below? Is it related to the core fix here, or just some extra stuff you fix when you discovered it?

Reply:
This is basically a fix which is needed here, but I did on main too (and I was a bit wrong). The point here is that when we open the settings screen, we should try to preserve the last selected configurable (as this screen is kind of a singleton).

`reloadConfigButtons` is called in `setSelected` which is called just below. We have to filter the new selected configurable because instances might have changed. 

So basically I only added new stuff here, Github diff is just a bit weird.

Reply:
Although I did discover a bug here.. I am fixing it.

Reply:
Ah, ok, it is about getting back to where we were before in the settings screen. I think I get it now.